src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_CompoundAssignmentOperator.cs

```diff
@@ -332,15 +332,30 @@ private BoundIndexerAccess TransformIndexerAccess(BoundIndexerAccess indexerAcce
                 indexerAccess.Type);
         }
 
-        private BoundFieldAccess TransformReferenceTypeFieldAccess(BoundFieldAccess fieldAccess, BoundExpression receiver, ArrayBuilder<BoundExpression> stores, ArrayBuilder<LocalSymbol> temps)
+        /// <summary>
+        /// Returns true if the <paramref name="receiver"/> was transformed.
```

Comment:
I think `transformed` in this spec should be `lowered and transformed`, and should give some hint as to how the method should decide whether or not it should return true or false.